### DISCUSSION TITLE
v1.5.7d

### DISCUSSION THREAD

**Discussion by zcwilt** [zencart/zencart#4646] [Announcements]
It is not meant for production. A production release will follow shortly

Patch release, including:

- Admin: Sanitize output
- Admin: Address missing/inconsistent HTML characteristics 
- Admin:  Remove call to inspectAndUpdate from every request and only call on plugin_manager page
- Admin: include plugins in search
 - Admin: Fix width of comments block in invoice
- Admin: Sanitize Output
- Admin: include plugins in search
- Admin: Sanitize Output
- Core: Fix typo: langauages -> languages
- Core: Accommodate field types with zero length
- Core: Fix "Uncaught TypeError: mysqli_close(): Argument 1 ($mysql) must be of type mysqli, bool given
- Core:  Fix coupon min. required calculation wrong when calculated from = All Products
- Storefront: Clean HTML for product_reviews_write, correcting PHP notices regarding missing variables.
- Core: paypaldp/paypal_curl, correcting PHP notices, partial payflow-uk correction.

(Note: no database changes between v157/v157a/v157b/v157c/v157d, so no need to run zc_install if v157/v157a/v157b/v157c is already installed.)

<hr /><em>This discussion was created from the release <a href='https://github.com/zencart/zencart/releases/tag/v1.5.7d'>v1.5.7d</a>.</em>

[Draft Zip Release](https://github.com/zencart/zencart/archive/refs/tags/v1.5.7d.zip)

**Comment by scottcwilson** [comment]
modified_files.txt is at the top level - this be removed.

**Reply by zcwilt** [reply]
yeah. forgot about that. it's an artefact created by the versionstamper

**Comment by scottcwilson** [comment]
The old "e-start your store" banner is still there in demo data. 
<img width="399" alt="banner" src="https://user-images.githubusercontent.com/4391638/155332469-38ae3294-a11a-4515-83ce-aab8a8c97f80.png">
